### DISCUSSION TITLE
How to setup ipv6 gateway?

### DISCUSSION THREAD

**Discussion by bb-lcloud** [DefGuard/defguard#803] [Q&A]
Hello  I want to ask how to setup ipv6 gateway.
My problem is that the screen in wchich you register new gateway in web UI doesn't allow to input ipv6
Related [docs(which screen I mean)](https://defguard.gitbook.io/defguard/admin-and-features/setting-up-your-instance/standalone-package-based-installation#gateway-the-wireguard-vpn-service)

I am entering ipv6 adress as gateway IP along with mask, but it says incorrect address.

I also tried using ipv6 from [ibm site](https://www.ibm.com/docs/en/ts3500-tape-library?topic=formats-subnet-masks-ipv4-prefixes-ipv6)

**Comment by kchudy** [comment]
Hi @bb-lcloud , entering IPv6 addresses was introduced in [v1.4](https://github.com/DefGuard/defguard/releases/tag/v1.4.0).
